prod/efd-kafka/terraform.tfvars

```diff
@@ -53,10 +84,15 @@ terragrunt = {
   } # terraform
 }
 
-dns_enable = true
 env_name = "prod"
-grafana_admin_pass = ""
-grafana_admin_user = ""
+gke_version = "1.11.7-gke.12"
```

Comment:
Is it necessary to not allow newer versions or is this a carry over from when gke was misbehaving?

Reply:
Agree, we don't need to pin the gke_version here.